app/controllers/statistics_controller.rb

```diff
@@ -12,6 +14,31 @@ def collection
     @all_indexers     = build_user_array(DeedType::PAGE_INDEXED)
   end
 
+  def export_csv
+    header = ['User Login', 'User Name', 'Email', 'Opt-In']
+    owner = User.find 'ushmmarchives'
+    collection_ids = owner.collections.map { |c| c.id }.sort
+    deed_map = Deed.where(:collection_id => collection_ids).group(:user_id, :collection_id).count
+    user_ids = deed_map.keys.map {|e| e[0]}.uniq
+    Collection.where(:id => collection_ids).order(:id).each { |c| header << c.title }
+
+    rows = []
```

Comment:
Initialize the rows -- these should contain one row per user, with four standard columns (email, opt-in, etc) and one column per collection.